fstar-helpers/minicore/Cargo.toml

```
@@ -0,0 +1,7 @@
+[package]
+name = "minicore"
+edition = "2021"
+publish = false
+
+[dependencies]
+rand = "0.9.0"
```

0.9.0 is not the latest.

```suggestion
rand = "0.9"
```